tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -131,7 +131,7 @@ def validate_output(self, result, cert_data):
         # validate correct installer used
         pattern, count = 'satellite-installer --scenario satellite', 2
         assert len(re.findall(pattern, result.stdout)) == count
-        pattern, count = 'foreman-installer --scenario katello', 0
```

Comment:
I believe this should not be changed, as this step/test validates if the correct installer is used, @omkarkhatavkar can confirm?

Reply:
it tests that there is 0 `katello` options, in installer, it doesn't matter which installer you are looking at

Reply:
@Gauravtalreja1 is correct, earlier we used to get a lot of issues with the cert generation command we wanted to ensure that foreman-installer options should not be seen. Please revert this.